agent/command/registry.go

```diff
@@ -34,6 +34,7 @@ func init() {
 		"generate.tasks":                generateTaskFactory,
 		"git.apply_patch":               gitApplyPatchFactory,
 		"git.get_project":               gitFetchProjectFactory,
+		"git.merge_pr":                  gitMergePrFactory,
```

Comment:
I think we should add a check to the validator that ensures that users can't use this command manually

Reply:
I think it's fine to call manually if there's ever a use case for this. There isn't anything dangerous here. Also I don't see that git.push has that validation and they should probably be consistent